### DISCUSSION TITLE
CH03_2 backend Django session 관련 질문

### DISCUSSION THREAD

**Discussion by ghost** [tedilabs/fastcampus-devops#24] [Q&A] [answered]
안녕하세요~, 혹시 CH03 에서 Django 파일 좀 받을 수 있을까요? 
CH03_2 4:03 이후 계속 'MultiValueDictKeyError' 가 떠서 진행이 안되네요

**Comment by posquit0** [comment, marked as answer]
안녕하세요! 해당 강의는 제 강의가 아니라 fastcampus에 문의 메일 보내셔서 담당 강사분과 연락을 취하셔야할 것 같습니다. :)
